test/tool/test_data/clamp_data.json

```diff
@@ -0,0 +1,344 @@
+{
+  "inputsData": {
+    "float64Data": [
```

Comment:
So this float64 data was cast to float32 first (and then back to float64) before passing to the baseline reference engine? Likewise, the data is also cast to float32 in the same way when passed to the web API implementation?

We just want to be sure that both inputs see the same initial tensor. If the tests are passing, then I guess they must be cast consistently.

Reply:
> So this float64 data was cast to float32 first (and then back to float64) before passing to the baseline reference engine? Likewise, the data is also cast to float32 in the same way when passed to the web API implementation?

Yes. The float64 data was cast to float32 by same invoking `new Float32Array(float64Data)`  before passing to baseline reference engine and web API implementation.

Reply:
Cool.